mlflow/diviner.py

```diff
@@ -192,6 +219,59 @@ def save_model(
     _PythonEnv.current().to_yaml(str(path.joinpath(_PYTHON_ENV_FILE_NAME)))
 
 
+def _save_model_fit_in_spark(diviner_model, path: str, **kwargs):
+    """
+    Saves a Diviner model that was fit in Spark by processing the model components separately.
+    The metadata and ancillary files to write (JSON and Pandas DataFrames) are written directly
+    to a fuse mount location, which the Spark DataFrame that contains the individual serialized
+    Diviner model objects is written by using the 'dbfs:' scheme path that Spark recognizes.
+    """
+    # Validate that the path is a relative path early in order to fail fast prior to attempting
+    # to write the (large) DataFrame to a tmp DFS path first and raise a path validation
+    # Exception within MLflow when attempting to copy the temporary write files from DFS to
+    # the file system path provided.
+    if not os.path.isabs(path):
+        raise MlflowException(
+            "The save path provided must be a run-relative path. "
+            f"The path submitted, '{path}' is an absolute path."
+        )
+
+    # Create a temporary DFS location to write the Spark DataFrame containing the models to.
+    tmp_path = generate_tmp_dfs_path(MLFLOW_DFS_TMP.get())
```

Comment:
`mlflow.spark.log_model()` & `mlflow.spark.save_model()` also include a `dfs_tmpdir` argument so that users can specify an alternative intermediate DFS location for the serialized model files. Can we add that to this API as well?

Reply:
Can we also add this argument to `load_model()`?

Reply:
great idea. added